db/people.yml

```diff
@@ -1642,7 +1642,7 @@ jliu:
       organization: Columbia University
       position: Post Doctoral Associate
   name: Jiwu Liu
-  position: Software Development Engineer
+  position: Post Doctoral Scholar
```

Comment:
Why did you change this?  I think it is correct before.  It should describe his current position if we know it.

Reply:
@sbillinge I am working on this with Songsheng on Zoom, it started failing tests after I did the rebase. He suggested changing the position but things are still wrong. I will fix it back to Software Development Engineer. He thinks the error is not from my part and suggesting you can merge it and he can do it later.